index.html

```diff
@@ -232,7 +232,7 @@ <h2 class="success__title">Изображение успешно загруже
     <div class="img-upload__message  img-upload__message--loading">Загружаем...</div>
   </template>
 
-  <script src='js/main.js'></script>
+  <script src='/js/main.js'></script>
```

Comment:
Если работаешь с модулями (а так нужно, это export/import) то нужно явно указать  это при подключении файла
```suggestion
  <script src='/js/main.js' type="module"></script>
```